lib/cli/src/Cardano/CLI.hs

```diff
@@ -106,6 +113,27 @@ parseArgWith cli args option = do
     getArgOrExit :: Arguments -> Option -> IO String
     getArgOrExit = getArgOrExitWith cli
 
+parseAllArgsWith :: FromText a => Docopt -> Arguments -> Option -> IO [a]
+parseAllArgsWith cli args option = do
+    (mapM (fromText . T.pack) <$> args `getAllArgsOrExit` option) >>= \case
+        Right a -> do
```

Comment:
strange how hlint doesn't react to this

Reply:
We've disable that one because, it's often quite practical to put some extra `do` (especially when working with hspec), and it doesn't cost us much. That's the kind of thing the compiler will take care of removing if it's an overhead. For us as developer, if it helps readability, I am all for having an extra `do` :)